app/views/users/_form.html.erb

```diff
@@ -161,8 +161,18 @@
       <script>
         (function() {
           $('.btn-save').click(function onClick(e) {
+          	var input1 = document.getElementById("username").value.length;
+          	var input2 = document.getElementById("email").value.length;
+          	var input3 = document.getElementById("password").value.length;
+          	var input4 = document.getElementById("password-confirmation").value.length;
+          	var input5 = document.getElementById("user_bio").value.length;
+          	if(input1 && input2 && input3 && input4 && input5) {
```

Comment:
Hi Rishabh, although the changes looks good but how about putting some styling. We use noty for generating alerts so instead of alert we can use noty for showing the notice - "username, password and Email are required fields, please complete them before proceeding." something like this. Also, there is no need to put check on user_bio. Thanks!

Reply:
Hey @gauravano, Nice suggestion, now the overall design looks better.
I've added :
  -noty.js for notificaitons
  -integrated noty.js with animate.css animations
  -added loops in javascript to show what all user inputs are empty